infra/ansible/vars/test.yml

```diff
@@ -26,6 +26,7 @@ services:
   - name: drc-test
     templates: ../k8s/drc/
     image_tag: develop
+    min_upload_size: 4294967296
```

Comment:
`4096m` is a bit more readable, but I'll have to check if you can use that for the envvar with uwsgi as well

Reply:
Please resolve this by adding a comment saying that this value represents: 4096m and be done with it.

Reply:
Yeah I checked with uwsgi options and just the number is enough.